src/_scss/pages/covid19/_stickyHeader.scss

```diff
@@ -10,55 +10,59 @@
             font-size: rem(21);
             @include align-self(flex-start);
         }
-        .fiscal-year-container {
-            @include display(flex);
-            @include flex-direction(column);
-            @include justify-content(center);
-            @include align-items(center);
-            padding-left: rem(12);
-            .usa-dt-picker {
-                background-color: #385C88;
-                height: rem(25);
-                width: 100%;
-                .usa-dt-picker__list-item {
-                    .usa-dt-picker__item {
-                        width: 100%;
-                        height: 100%;
-                    }
-                    .active {
-                        background-color: #f1f1f1;
-                        color: #0071bc;
-                    }
-                }
-                .usa-dt-picker__dropdown-container {
-                    background-color: #385C88;
-                    margin: auto;
-                }
-                .usa-dt-picker__button-text,
-                .usa-dt-picker__button-icon svg {
-                    color: $color-white;
-                }
-                .usa-dt-picker__icon svg {
-                    width: rem(14);
-                    margin-left: rem(10);
-                }
-                .usa-dt-picker__button {
-                    padding: 0 rem(10);
-                    margin: 0;
-                }
-                .usa-dt-picker__button-text {
-                    height: rem(23);
-                    width: rem(37);
-                    font-size: rem(18);
-                    font-weight: 600;
-                    letter-spacing: 0;
-                    line-height: rem(23);
-                    margin: 0;
-                }
-                .usa-dt-picker__button-icon {
-                    padding-left: rem(10);
-                }
-            }
+        // Not used ATM
```

Comment:
can we keep comments with our standard form `// TODO - Uncomment this when fiscal years selector is implemented`?